superduperdb/db/base/db.py

```diff
@@ -85,6 +86,11 @@ def __init__(
         self.databackend = databackend
         self.vector_database = vector_database
         self._distributed_client = distributed_client
+        self._cdc = DatabaseChangeDataCapture(self)
+
+    @property
```

Comment:
Could be a `cached_property`?

Reply:
Or simply set-up at start

Reply:
E.g. `self.cdc = DatabaseChange...` don't need `@property`.

Reply:
Should be something like:

`if s.CFG.cdc: self._cdc = DataBaseChangeDataCapture(...)`